packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -544,7 +545,15 @@ export class ModularChangeFamily
 				field.change = fieldKind.changeHandler.rebaser.rebase(
 					fieldChangeset,
 					tagChange(baseChangeset, context.baseRevision),
-					(node) => node,
+					(curr, base) => {
+						if (curr === undefined) {
+							return base === undefined
+								? undefined
+								: crossFieldTable.rebasedNodeCache.get(base);
+						} else {
+							return crossFieldTable.rebasedNodeCache.get(curr) ?? curr;
```

Comment:
Why do we return `curr` if the `rebasedNodeCache` entry is undefined? That seems incorrect.

Reply:
I was thinking in terms of "this is a case where the cache is not changing the behavior of the old code" but it didn't occur to me that the old code may not be correct in this case.
What's an example of a sitation where that's not what we want?
I'm guessing we should be performing the normal NodeChangeset rebasing in this case, is that right?

Reply:
In the current implementation the rebasedNodeCache entry should always be defined, so I guess there's no example where this is wrong. But we shouldn't need to do anything special here other than potentially asserting that the entry is defined.